apicapi/apic_client.py

```diff
@@ -335,6 +335,27 @@ class ManagedObjectClass(object):
                 'rsvswitchExporterPol-[uni/infra/vmmexporterpol-%s]'),
         'netflowVmmExporterPol': ManagedObjectName('infraInfra',
                                                    'vmmexporterpol-%s'),
+
+        # ERSPAN
+        'spanVSrcGrp': ManagedObjectName('infraInfra', 'vsrcgrp-%s'),
+        'spanVSrcGrp': ManagedObjectName('fvTenant', 'vsrcgrp-%s'),
+        'spanSpanLbl': ManagedObjectName('spanVSrcGrp', 'spanlbl-%s'),
+        'spanVSrc': ManagedObjectName('spanVSrcGrp', 'vsrc-%s'),
+        'spanRsSrcToVPort':
+        ManagedObjectName(
+                'spanVSrc',
+                'rssrcToVPort-[uni/tn-%(name)s/ap-%s/epg-%s/cep-%s]'),
+        'fvCEp': ManagedObjectName('fvAEPg', 'cep-%s'),
```

Comment:
After talking with Tom, I don't think we need the fvCEp MO. ERSPAN references an fvCEp, but we'll never need to configure it in OpenStack.

Reply:
Removed it in the latest commit